static/js/compose_state.js

```diff
@@ -5,6 +5,10 @@ import * as compose_pm_pill from "./compose_pm_pill";
 let message_type = false; // 'stream', 'private', or false-y
 let recipient_edited_manually = false;
 
+// We use this variable to keep track of whether user has viewed the topic resolved
+// banner - so that we can show it to them only once
```

Comment:
Can you add a bit more specificity here? We specifically want to show it only once per continuous time of composing in a particular topic, however we can more clearly describe that heh.